miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -436,6 +436,18 @@ private static DetailedSample getParent(DetailedSampleDto childDto) {
         tissue.getSampleClass().setId(childDto.getParentTissueSampleClassId());
         tissue.setParent(parent);
         parent = tissue;
+
+        if (childDto instanceof SampleLCMTubeDto) {
+          SampleLCMTubeDto lcm = (SampleLCMTubeDto) childDto;
+          if (lcm.getParentSlideClassId() != null) {
+            SampleSlide slide = new SampleSlideImpl();
+            slide.setSampleClass(new SampleClassImpl());
+            slide.getSampleClass().setId(lcm.getParentSlideClassId());
+            slide.setSlides(0);
```

Comment:
Should be `lcm.getSlidesConsumed()` otherwise the ghost parent slide has negative slides remaining.

Reply:
fixed